lib/openapi3.js

```diff
@@ -77,6 +77,10 @@ function getTagGroup(tag, tagGroups){
 }
 
 function fakeProdCons(data) {
+    const apiTarget = data.url.split('/')[4];
+    data.authParameter = {};
+    data.authParameter = apiTarget === "hackers" ? (
+        {username:"username",token:"+RkfaGmptinbuqtURvzscSf/Sm4NXlw/x9pPtaLLzLE="}) : ({username:"api_example_company",token:"Ke+2jinhe5jM87P95aAVOz7L3ZWrtSiERtyOkkh5tEQ="});
```

Comment:
FYI for the Hacker API  are fake credentials, for the customer API the existing sandbox credentials

Reply:
LGTM! @martijnrusschen shall we rename `api_example_company` to `api_example_identifier` for Customer API based on [this comment](https://hackerone.slack.com/archives/C042GJ0U1/p1623256086005200)?

Reply:
The `api_example_company` is linked to an actual API token that you can use to test requests. I think it's fine to keep it as is.